tests/client.c

```diff
@@ -333,6 +218,9 @@ send_request_and_read_response(struct demo_conn *conn,
   fd_set read_fds;
   fd_set write_fds;
   size_t n = 0;
+  const char *body, *content_length_str, *content_length_end;
```

Comment:
nit
```suggestion
  const char *body; 
  const char *content_length_str;
  const char *content_length_end;
```
...would be my preference but feel free to ignore me.